Core/PrimaryGenerator.cpp

```diff
@@ -218,7 +218,7 @@ void PrimaryGenerator::GenerateEvtSmallChamber(
     //! oPs 3G
     event->AddPrimaryVertex(GenerateThreeGammaVertex(
       vtxPosition, T0,
-      material->GetLifetime(random - evtFractions[1] - evtFractions[2] - evtFractions[3] - evtFractions[4], MaterialExtension::DecayChannel::Ortho3G)
+      material->GetLifetime(random - evtFractions[0] - evtFractions[1] - evtFractions[2] - evtFractions[3], MaterialExtension::DecayChannel::Ortho3G)
```

Comment:
Can you comment what was the reason for these changes and in general the changes from commit 
3f741f3 ?

Reply:
Probably I commented this mistake in Jyoti`s commit. I figured out when I tested lifetime distributions that I made a mistake previously. It should be as changed here.
I corrected it here because Sushil wanted to start simulations with easier selection of different processes, as introduced in this PR.
Other change was adding flag for saving decayTree, so useful one.

Reply:
Ok, thanks.